src/server/config.py

```diff
@@ -1,21 +1,30 @@
 import os
 import sqlalchemy as db
+from sqlalchemy_utils import database_exists, create_database
 
 # Determine if app is ran from docker or local by env var IS_LOCAL
 BASE_PATH = '/app/static/'
-DB = 'postgresql://postgres:thispasswordisverysecure@paws-compose-db/postgres'
+POSTGRES_PASSWORD=os.getenv('POSTGRES_PASSWORD', 'thispasswordisverysecure')
+POSTGRES_DATABASE=os.getenv('POSTGRES_DATABASE', 'paws')
+POSTGRES_USER=os.getenv('POSTGRES_USER', 'postgres')
+DB = 'postgresql://' + POSTGRES_USER + ':'+ POSTGRES_PASSWORD + '@paws-compose-db/' + POSTGRES_DATABASE
 
```

Comment:
Much better than having it hard-coded. What about just simplifying to a single config point of a connection string? Then in theory we could use whatever database technology we wanted and only need to read in one config.